jobs/build/github-ldap-mapping-update/README.md

```diff
@@ -0,0 +1,3 @@
+# github-ldap-mapping-update
+
+github-ldap-mapping-update extracts the mapping `m(github_login)=kerberos_id` from Red Hat LDAP server and exports it to a configMap on `app.ci`.
```

Comment:
Can you add the instructions you provided me about how the binary was extracted & how the kubeconfig was created.

Reply:
Done.!